src/app/assets/styles/_color_variables.scss

```diff
@@ -16,3 +16,5 @@
 $primary: #326DE6;
 $line: #AAAAAA;
 $subline: #888888;
+$link: #326DE6;
```

Comment:
Do we actually need to redefine primary color? I think primary is generic enough.